perceval/backends/core/discourse.py

```diff
@@ -51,17 +54,25 @@ class Discourse(Backend):
     :param api_token: Discourse API access token
     :param tag: label used to mark the data
     :param archive: archive to store/retrieve items
+    :param max_retries: number of max retries to a data source
+        before raising a RetryError exception
+    :param sleep_time: time to sleep in case
```

Comment:
Can we define whether this is milisecs, seconds, minutes, etc? Same below in client docs.

Reply:
I'm going to address this in another PR since the description of `sleep_time` is repeated in several backends.